about.html

```diff
@@ -22,24 +21,19 @@
 <h3>Applications</h3>
 
 <ul>
```

Comment:
All of these should use em  dashes, not hyphens.

Reply:
It hurts my eyes, I disagree here.

![schermata-11-29-17-42](https://user-images.githubusercontent.com/10681413/143907966-5540c4b1-8359-44de-8e22-1e41fbf9bc34.png)

Reply:
Em dashes are correct, logical order, naming doesn't seem out of order by putting "lxqt" next to "LXQt".
The second column could be aligned, but those are minute problems.